sparkpost/django/email_backend.py

```diff
@@ -0,0 +1,79 @@
+from django.core.mail.backends.base import BaseEmailBackend
+from sparkpost import SparkPost
+from django.conf import settings
+from .exceptions import UnSupportedContent, UnSupportedParam
+
```

Comment:
for imports let's follow this convention:

- stdlib imports first, in alphabetical order, newline after
- library imports (e.g. pytest, django, mock) next, in alphabetical order, newline after
- project-specific imports (e.g. sparkpost.something) next, in alphabetical order